src/tools/sign-oci-layer-root-hashes/src/registry.rs

```diff
@@ -366,7 +367,8 @@ async fn create_decompressed_layer_file(
             .await
             .context("Failed to pull layer file")?,
     }
-    attach_tarfs_index(decompressed_path).context("Failed to attach tarfs index")?;
+    // attach_tarfs_index(decompressed_path).context("Failed to attach tarfs index")?;
```

Comment:
Please delete the no longer relevant lines.

Reply:
Removed those lines